### PR TITLE
[vSphere] Add and update Dashboards

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## Description
Add New dashboards
Update existing dashboards

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Screen Shots
Screen shots has been added to integration, please review from integration files.

## Related issues
- Closes https://github.com/elastic/obs-integration-team/issues/91

### CODE REVIEW

File: packages/vsphere/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.16.3-next"
+  changes:
+    - description: Add and Update dashboards.
```
Comment (harnish-elastic) [inline packages/vsphere/changelog.yml:4]:
```suggestion
    - description: Add new dashboards and update existing dashboards.
```

File: packages/vsphere/img/vsphere_cluster_network_dashboard.png
Comment (harnish-elastic) [inline packages/vsphere/img/vsphere_cluster_network_dashboard.png:1]: Minor nit: Update image with the dashboard title!

File: packages/vsphere/img/vsphere_resourcepool_dashboard.png
Comment (harnish-elastic) [inline packages/vsphere/img/vsphere_resourcepool_dashboard.png:1]: Minor nit: Update image with the dashboard title!

**Review by harnish-elastic — APPROVED** [review]
Minor nit. Otherwise LGTM!

File: packages/vsphere/manifest.yml [hunk 2]
```diff
@@ -9,18 +9,26 @@ categories:
   - virtualization
 conditions:
   kibana:
-    version: "^8.15.2"
+    version: "^8.16.2"
```
Comment (harnish-elastic) [inline packages/vsphere/manifest.yml:12]: I think the beats id changes will be available from 8.17.0 version. So can you change this to 8.17.0?

Comment (kush-elastic) [reply packages/vsphere/manifest.yml:12]: No, we are going to have this changes in `8.16.2` only.

**Comment by ishleenk17** [comment]
Review for Host Dashboard:

- This looks to be an overview dashboard as we have host, datatstore, network related visualzations.
- There are visualizations that use the word "stacked" in the title. We don't need to use the word while using stacked visualization
- Host activity over time: What activity are we showing here ? Whats the count in this chart ?
- In datastore types, I don't see the types listed on the right side of the chart ?
- We should not have [metrics vsphere] in title of every panel. Any reason why it us there ?


Will be reviewing others as well in next comments. Please note, I am taking a look at screenshots directly for review as thats the best way to review dashboard changes.

**Comment by ishleenk17** [comment]
Datastore dashboard review:

Y axis: capacity utilization % and capcity
2 panels have same title. Change that

**Comment by ishleenk17** [comment]
Resourcepool dashboard review:
1. Private vs shared memory usage will show for all the resourcepools ? Won't the graph get chaotic ?

Virtualmachine dashboard review:
1. VM's placement on ESXI hosts: Too many VM's get listed. We can list the top 10, IMO.
2. VM's per ESXI host: Does that tell about the number of VM's per EXI host? The screenshot doesn't give clarity
3. Networks from virtualmachine should be renamed as Networks associated with Virtualmachine

**Comment by ishleenk17** [comment]
The cluster and network are in the same dashboard. Let's give a horizontal divisioning rather than vertical divisioning between cluster and network.

**Review by ishleenk17 — APPROVED** [review]
Changes look good now.
Let's get the CI passing and we are good to merge then

**Pull request merged**